src/detect-transform-base64.c

```diff
@@ -120,6 +120,10 @@ static void TransformFromBase64Decode(InspectionBuffer *buffer, void *options)
     const uint32_t input_len = buffer->inspect_len;
     uint32_t decode_length = input_len;
 
+    if (input_len == 0) {
+        return;
+    }
```

Comment:
It looks better indeed but where do we create the variable length array?

Reply:
`uint8_t decoded[decoded_size];` below